crates/telio-nurse/src/heartbeat.rs

```diff
@@ -262,6 +264,18 @@ impl Runtime for Analytics {
     type Err = ();
 
     async fn wait(&mut self) -> WaitResponse<'_, Self::Err> {
+        let derp_event_future = if let Some(channel) = self.io.derp_event_channel.as_mut() {
```

Comment:
would'nt this mean, that we will not receive anything from multiplexer until we have received DERP event?